index.html

```diff
@@ -234,17 +235,17 @@ <h3>Interlinear notes</h3>
       </section>
 
       <section>
-        <h3>Ruby annotation to indicate the reading of a foreign phrase in language textbooks</h3>
+        <h3>Ruby annotations to indicate the reading of a foreign phrase in language textbooks</h3>
         <p>
-          In language textbooks, ruby annotation is sometimes used to indicate the reading of a foreign phrase in hiragana or katakana. 
-          For example, a Chinese phrase <span lang="zh-hans">我去学校</span> may have <span lang="ja">ウオ チュー シュエシャオ</span> as ruby annotation.
+          In language textbooks, ruby annotations are sometimes used for indicating the reading of a foreign phrase in hiragana or katakana. 
+          For example, a Chinese phrase <span lang="zh-hans">我去学校</span> may have <span lang="ja">ウオ チュー シュエシャオ</span> as a ruby annotation.
         </p>
       </section>
 
       <section>
         <h3>Double-sided Ruby</h3>
         <p>
-          A sequence of base characters may be accompanied by two ruby annotations. 
+          A sequence of characters may be accompanied by two ruby annotations. 
           Typically, one of them is [=Furigana=] and the other is either a [=GIKUN=] or [=interlinear note=].  
           In <a data-cite="JLREQ#fig2_3_12">an example in JLreq</a> 
           ("An example of ruby attached to both sides of the base characters"), 
```

Comment:
this `base characters` is also converted to e.g. `characters of ruby base` or something (just `ruby base`?)?